src/main/java/com/fixadate/global/jwt/service/JwtProvider.java

```diff
@@ -135,7 +140,8 @@ public boolean validateToken(String token) {
 	public UsernamePasswordAuthenticationToken getAuthentication(String token) {
 		String id = getIdFromToken(token);
 		Member member = memberRepository.findMemberById(id).orElseThrow(() ->
-			new TokenException(NOT_FOUND_MEMBER_IDENTIFIER));
+																			new TokenException(
+																				NOT_FOUND_MEMBER_IDENTIFIER));
```

Comment:
```suggestion
Member member = memberRepository.findMemberById(id)
										.orElseThrow(() -> new TokenException(NOT_FOUND_MEMBER_IDENTIFIER));
```

개행이 위와 같이 이루어지면 좋겠네요!
(변경 외 부분이 있어 블록이 잡히지 않아 요상하게 선택됐습니다 ... 🥲)

Reply:
너무 길어져 하기와 같이 변경하겠습니다!

```
		Member member = memberRepository.findMemberById(id).orElseThrow(
			() -> new TokenException(NOT_FOUND_MEMBER_IDENTIFIER)
		);
```